pubspec.yaml

```diff
@@ -25,4 +27,6 @@ flutter:
     platforms:
       android:
         package: com.truecallersdk
+        pluginClass: TruecallerSdkPlugin
+      ios:
         pluginClass: TruecallerSdkPlugin
```

Comment:
shouldn't this be `SwiftTruecallerSdkPlugin.swift`?

Reply:
Nope, the objective c class in turn invokes the swift plugin, so this should work